pypit/artrace.py

```diff
@@ -999,6 +1006,9 @@ def trace_slits(slf, mstrace, det, pcadesc="", maskBadRows=False):
     else: # There's an order overlap
         rsub = edgbtwn[1]-(lval)
     """
+    if msgs._debug['trace']:
+        from xastropy.xutils import xdebug as xdb
```

Comment:
gone. I was probably using xdb to do some plotting
(now no longer needed). 

All of xastropy should be gone now.